esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3277,4 +3326,586 @@ void WsWuHelpers::checkAndTrimWorkunit(const char* methodName, StringBuffer& inp
 
     return;
 }
+
+IFileIOStream* CWsWuFileHelper::createIOStreamWithFileName(const char* fileNameWithPath, IFOmode mode)
+{
+    Owned<IFile> wuInfoIFile = createIFile(fileNameWithPath);
+    Owned<IFileIO> wuInfoIO = wuInfoIFile->open(mode);
+    if (!wuInfoIO)
+        throw MakeStringException(ECLWATCH_CANNOT_OPEN_FILE, "Failed to open %s.", fileNameWithPath);
+    return createIOStream(wuInfoIO);
+}
+
+void CWsWuFileHelper::writeToFile(const char* fileName, size32_t contentLength, const void* content)
+{
+    if (isEmptyString(fileName))
+        throw MakeStringException(ECLWATCH_CANNOT_COMPRESS_DATA, "File name not specified.");
+    Owned<IFile> wuInfoIFile = createIFile(fileName);
+    Owned<IFileIO> wuInfoIO = wuInfoIFile->open(IFOcreate);
+    if (wuInfoIO)
+        wuInfoIO->write(0, contentLength, content);
```

Comment:
Should it be logged or some type of alert if this fails?

Reply:
Ok, Will do.